src/darwin/policy-softener-glue.c

```diff
@@ -0,0 +1,48 @@
+#include "frida-helper-backend.h"
+
+#ifdef HAVE_IOS
+#include <mach/mach.h>
+#endif
+
+extern kern_return_t bootstrap_look_up (mach_port_t  bootstrap_port, char * service_name, mach_port_t* service_port);
+typedef int (* JbdCallFunc) (guint32 service_port, guint command, guint pid);
+
+guint32
+_frida_electra_policy_softener_internal_jb_connect ()
+{
+#ifdef HAVE_IOS
```

Comment:
All of these functions should be part of the `#ifdef HAVE_IOS` at the top, as the Vala class declaring them is inside `#if IOS`.